doc/how_to_guides/isaac_panda/docker-compose.yaml

```diff
@@ -0,0 +1,44 @@
+# Docker Compose file for Isaac ROS2 Control
+#
+# Usage:
+#
+# To build the images:
+#   docker compose build
+#
+# To start up a specific service by name:
+#   docker compose up <name>
+#
+# To open an interactive shell to a running container:
+# (You can tab complete to get the container name)
+#   docker exec -it <container_name> bash
+
+services:
+  base:
+    image: isaac_moveit_tutorial
+    build:
+      context: .
+      dockerfile: .docker/Dockerfile
+    # Interactive shell
+    stdin_open: true
+    tty: true
+    # Networking and IPC for ROS 2
+    network_mode: host
+    ipc: host
+    privileged: true
+    environment:
+      - ROS_DOMAIN_ID
+      # Allows graphical programs in the container
+      - DISPLAY=${DISPLAY}
+      - QT_X11_NO_MITSHM=1
+      - NVIDIA_DRIVER_CAPABILITIES=all
+      - NVIDIA_VISIBLE_DEVICES=all
+    volumes:
+      # Allows graphical programs in the container
+      - /tmp/.X11-unix:/tmp/.X11-unix:rw
+    # Allows GPU access -- comment out this block if you don't have one
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - capabilities: [gpu]
```

Comment:
If a user does not have a GPU on their system, this will cause `docker compose` commands to fail. 

I'd say just remove it if we're not running anything GPU-ey inside the container, or you could do what they do here and have separate services https://github.com/MarqRazz/moveit2_tutorials/blob/pr-isaac_tutorial/.docker/docker-compose.yml

Reply:
@sea-bass can we extend the service to have a gpu supported version? I think most users of Isaac-sim will have a GPU installed as it is a system requirement (https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/requirements.html).

Reply:
We probably should, but I'm not worried about it since the container is only running MoveIt (and it doesn't really matter if that has a GPU available) and the host is running Isaac Sim.

Reply:
Yeah my apologies I went to view file and deleted my comment when I realised this. If only running MoveIt it makes sense to not include the above in the docker-compose file.